go/vt/discovery/healthcheck.go

```diff
@@ -168,6 +172,7 @@ func registerWebUIFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&TabletURLTemplateString, "tablet_url_template", "http://{{.GetTabletHostPort}}", "Format string describing debug tablet url formatting. See getTabletDebugURL() for how to customize this.")
 	fs.DurationVar(&refreshInterval, "tablet_refresh_interval", 1*time.Minute, "Tablet refresh interval.")
 	fs.BoolVar(&refreshKnownTablets, "tablet_refresh_known_tablets", true, "Whether to reload the tablet's address/port map from topo in case they change.")
+	fs.Int64Var(&healthCheckConnectConcurrency, "healthcheck-connect-concurrency", 1024, "Concurrency of new healthcheck connections. This should be less than the golang max thread limit of 10000.")
```

Comment:
An alternative: `--healthcheck-dial-concurrency` 🤔

An some feedback on the `1024` default: it's working nicely for us without introducing any noticeable delays to startup or hitting the 10k thread limit

Reply:
Updated to `--healthcheck-dial-concurrency`. It explains what is going on better 👍